test_conformance/extensions/cl_khr_command_buffer/negative_command_buffer_fill.cpp

```diff
@@ -416,6 +416,13 @@ struct CommandBufferCommandFillBufferMutableHandleNotNull
 {
     using CommandFillBaseTest::CommandFillBaseTest;
 
+    bool Skip() override
+    {
+        if (CommandFillBaseTest::Skip()) return true;
+        return is_extension_available(device,
+                                      "cl_khr_command_buffer_mutable_dispatch");
```

Comment:
The mutable-dispatch extension affects kernel commands, not other types of commands. The spec wording for [clCommandFillBufferKHR](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_API.html#clCommandFillBufferKHR) and other non-kernel command appending entry-points is:

> * mutable_handle returns a handle to the command. This parameter is unused, and must be NULL."

The "must be NULL" part isn't conditional on the support for `cl_khr_command_buffer_mutable_dispatch`, so I don't think skipping here is correct. However, a future [cl_khr_command_buffer_mutable_memory_commands](https://github.com/KhronosGroup/OpenCL-Docs/pull/1065) extension could relax this for other types of commands.

Reply:
You're right, my mistake. I've dropped the commit.